pkg/columns/formatter/textcolumns/options.go

```diff
@@ -37,6 +37,7 @@ type Options struct {
 	DefaultColumns []string    // defines which columns to show by default; will be set to all visible columns if nil
 	HeaderStyle    HeaderStyle // defines how column headers are decorated (e.g. uppercase/lowercase)
 	RowDivider     string      // defines the (to be repeated) string that should be used below the header
+	DoEllipsis     bool        // defines whether to add or not ellipsis.
```

Comment:
Maybe `TruncateStrings` would be a better and more generic fit, because right now this defines whether or not to shorten the strings, not whether an ellipsis will be used.